asmcomp/cmm_helpers.mli

```diff
@@ -206,7 +206,8 @@ val field_address : expression -> int -> Debuginfo.t -> expression
 (** [get_field_gen mut ptr n dbg] returns an expression for the access to the
     [n]th field of the block pointed to by [ptr] *)
 val get_field_gen :
-  Asttypes.mutable_flag -> expression -> int -> Debuginfo.t -> expression
+  ?memory_chunk:memory_chunk -> Asttypes.mutable_flag -> expression -> int ->
+  Debuginfo.t -> expression
```

Comment:
I think the ideal parameter order would be to have `ptr` first, as `memory_chunk` and `mut` rather depend on the offset, but this is okay. Oh, could you maybe mention the parameter in the documentation comment?

Reply:
I kept the order of the constructor parameters. If you think this should be put elsewhere I'm ok with it.